x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list.tsx

```diff
@@ -0,0 +1,422 @@
+              </EuiText>
+            </EuiToolTip>
+          );
+        },
+      },
+      // conditional hostname column
+      {
+        field: 'logEntries',
+        name: TABLE_COLUMN_NAMES.comments,
+        width: '30%',
+        truncateText: true,
+        render: (logEntries: ActionDetails['logEntries']) => {
+          const comment =
+            (logEntries.filter((e) => e.type === 'action')[0] as EndpointActivityLogAction).item
+              .data.EndpointActions.data.comment || '-';
+          return (
+            <EuiToolTip content={comment} anchorClassName="eui-textTruncate">
+              <EuiText
+                size="s"
+                className="eui-textTruncate eui-fullWidth"
+                data-test-subj={getTestId('column-comments')}
+              >
+                <p className="eui-displayInline eui-TextTruncate">{comment}</p>
+              </EuiText>
+            </EuiToolTip>
+          );
+        },
+      },
+      {
+        field: 'isCompleted',
+        name: TABLE_COLUMN_NAMES.status,
+        width: '10%',
+        render: (isCompleted: ActionDetails['isCompleted'], data: ActionDetails) => {
+          const status = data.isExpired
+            ? UX_MESSAGES.badge.failed
+            : isCompleted
+            ? data.wasSuccessful
+              ? UX_MESSAGES.badge.completed
+              : UX_MESSAGES.badge.failed
+            : UX_MESSAGES.badge.pending;
+
+          return (
+            <EuiToolTip content={status} anchorClassName="eui-textTruncate">
+              <EuiBadge
+                data-test-subj={getTestId('column-status')}
+                color={
+                  data.isExpired
+                    ? 'danger'
+                    : isCompleted
+                    ? data.wasSuccessful
+                      ? 'success'
+                      : 'danger'
+                    : 'warning'
+                }
+              >
+                <FormattedMessage
+                  id="xpack.securitySolution.responseActionsList.list.item.status"
+                  defaultMessage="{status}"
+                  values={{ status }}
+                />
+              </EuiBadge>
+            </EuiToolTip>
+          );
+        },
+      },
+      {
+        field: '',
+        align: RIGHT_ALIGNMENT as HorizontalAlignment,
+        width: '40px',
+        isExpander: true,
+        name: (
+          <EuiScreenReaderOnly>
+            <span>{UX_MESSAGES.screenReaderExpand}</span>
+          </EuiScreenReaderOnly>
+        ),
+        render: (data: ActionDetails) => {
+          return (
+            <EuiButtonIcon
+              onClick={() => toggleDetails(data)}
+              aria-label={itemIdToExpandedRowMap[data.id] ? 'Collapse' : 'Expand'}
+              iconType={itemIdToExpandedRowMap[data.id] ? 'arrowUp' : 'arrowDown'}
+            />
+          );
+        },
+      },
+    ];
+    if (hideHostNameColumn) {
```

Comment:
Yeah, that makes sense. This would be easier to filter when each field is unique. I'm using logEntries for a couple to get comments and parameters. I'll mark this to do once we get the API changes in.